### PR TITLE
Added noninteractive for apt-get installs

### PR DESCRIPTION
**Pull request by BlackthornYugen**
This prevents a lot of the error messages during install that begin with "debconf: unable to initialize frontend: Dialog". 
screen shot 2015-05-10 at 1 30 43 pm
This likely doesn't make any functional difference but it would make it easier to spot actual problems during a docker build.

### CODE REVIEW

**Comment by fideloper** [comment]
Howdy!
Looks like a lot of your commits are coming through here, some specific to your repository instead of this one!

(However I'd love to pull in the noninteractive bit!)

**Comment by BlackthornYugen** [comment]
Sorry about that, this was my first cross-repo pull request. I guess I did something wrong. Let me try again.

**Comment by fideloper** [comment]
No problem! (I think it keeps adding commits as you make them to the branch
used for the PR, I forget if there's a way to stop that other than making a
branch specifically to PR from)

**Comment by BlackthornYugen** [comment]
Right, thats what it looks like. I wonder if I do that and then deleted the pulled branch, would it mess anything up?

**Comment by fideloper** [comment]
I'm ~90% sure that'll work fine

**Comment by BlackthornYugen** [comment]
Yup! #19 still shows that it was pulled from the branch even after it was deleted. I thought _maybe_ it might say that it pulled from master or possibly stop tracking the history after deleting the branch.

**Comment by fideloper** [comment]
Glad it didn't, github is pretty snazzy !
